webpack.config.js

```diff
@@ -1,16 +1,52 @@
 var path = require('path');
+var webpack = require('webpack');
 
+var isDev = process.env.NODE_ENV !== 'production';
 
 module.exports = {
-    entry: [
-      'webpack/hot/dev-server', path.resolve(__dirname, 'app/index.js')],
-    module: {
-      loaders: [{
-        loader: 'babel'
-      }]
-    },
-    output: {
-        path: path.resolve(__dirname, 'build'),
-        filename: 'bundle.js',
-    }
+
+  devtool: isDev ? 'eval' : 'source-map',
+
+  entry: [
+    path.join(__dirname, 'src/main.js')
+  ].concat(isDev ? [
+    'webpack-dev-server/client?http://localhost:8000',
+    'webpack/hot/only-dev-server'
+  ] : []),
+
+  output: {
+    path: 'src/build/',
+    filename: '[name].js',
+    publicPath: 'build/'
```

Comment:
questo è usato quando sviluppi, il file non è veramente scritto su disco, ma webpack lo tiene in memoria